framework/src/application/node/synchronizer/synchronizer.js

```diff
@@ -15,26 +15,34 @@
 'use strict';
 
 const assert = require('assert');
+const { validator } = require('@liskhq/lisk-validator');
+const { Status: TransactionStatus } = require('@liskhq/lisk-transactions');
+const definitions = require('../schema/definitions');
 const utils = require('./utils');
 
 class Synchronizer {
 	constructor({
+		channel,
 		logger,
 		blocksModule,
 		processorModule,
 		storageModule,
+		transactionPoolModule,
 		mechanisms = [],
 	}) {
 		assert(
 			Array.isArray(mechanisms),
 			'mechanisms should be an array of mechanisms',
 		);
 		this.mechanisms = mechanisms;
+		this.channel = channel;
 		this.logger = logger;
 		this.blocksModule = blocksModule;
 		this.processorModule = processorModule;
+		this.transactionPoolModule = transactionPoolModule;
 		this.storageModule = storageModule;
 		this.active = false;
+		this.loadTransactionsRetries = 5;
```

Comment:
could we make this a constant?

Reply:
Do you mean that this constant should come from config?